configs/all-bridged-tokens.json

```diff
@@ -9,7 +9,13 @@
     "source": "Ethereum",
     "bridge": "ForceBridge",
     "ckbExplorerUrl": "https://explorer.nervos.org/sudt/0x9623b3027c4718cab7b0b41521808bfc4a94a855638b20f5a5dc92e7e89b3234",
-    "ckbL2ContractAddress": ""
+    "ckbL2ContractAddress": "",
+    "uanInfo": {
+      "sourceUan": "ETH.eth",
+      "sourceDisplayName": "ETH",
+      "bridgedUan": "ETH.ckb|fb.eth",
+      "bridgedDisplayName": "ETH (via Forcebridge from ETH)"
```

Comment:
maybe we should use "Force Bridge" to avoid a non-uniform product name here?

Reply:
> maybe we should use "Force Bridge" to avoid a non-uniform product name here?

renamed